### PR TITLE
Added WHL and CFL platform support documentation

### PR DESCRIPTION
**Pull request by raghavag**
This guide documents platform support enabled for
Whiskey Lake, Coffee Lake Refresh Desktop S8+2 and
Coffee Lake Refresh Mobile H6+2 platforms

Signed-off-by: Raghava Gudla <raghava.gudla@intel.com>

### CODE REVIEW

File: source/supported-hardware/whl-cfl-crb.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  |           |CFLHR|         |    **J4A1**  |
+  +---------------------------+--------------+
+
+Booting Yocto Linux from USB
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You may need to change boot options to boot from USB. See :ref:`change-boot-options`.
+
+1. Download Yocto Linux
+2. Create bootable USB key. For example: In Windows, roofus can be used. In Linux, etcher app can be used.
```
Comment (mauricema) [inline source/supported-hardware/whl-cfl-crb.rst]: rootfus ?

File: source/supported-hardware/whl-cfl-crb.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+    python Platform/CoffeelakeBoardPkg/Script/StitchLoader.py -i xxxx.bin -s Outputs/cfl/Stitch_Components.zip -o sbl_cfl_ifwi.bin
+
+For more details on stitch tool, see :ref:`stitch-tool` on how to stitch the IFWI image with |SPN|.
+
+
+Flashing
+^^^^^^^^^
+
+Flash the generated ``sbl_cfl_ifwi.bin`` to the target board using DediProg SF100 or SF600 programmer.
```
Comment (mauricema) [inline source/supported-hardware/whl-cfl-crb.rst]: Add info on the flash offset to flash.   I assume 0.

File: source/supported-hardware/whl-cfl-crb.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  |           |CFLHR|         |    **J7H2**  |
+  +---------------------------+--------------+
+
+.. note:: Please disconnect Deidprog before powering up the board again.
+
+
+Debug UART
+^^^^^^^^^^^
+
+Serial port connector for CRB platforms is provided in the table below
```
Comment (mauricema) [inline source/supported-hardware/whl-cfl-crb.rst]: Add one line to mention the UART settings , like 115200N8

File: source/how-tos/boot-ubuntu.rst [hunk 4]
```diff
@@ -5,26 +5,10 @@ Boot Ubuntu
 
 |SPN| can boot Ubuntu Linux using |SPN| with default OS loader. This page provides a step-by-step guide how to do so.
 
-The following steps have been verified with Ubuntu Linux 18.04 LTS (64-bit) on |UP2| board.
```
Comment (mauricema) [inline source/how-tos/boot-ubuntu.rst]: Maybe good to leave it has been verified  with Ubuntu Linux 18.04 LTS (64-bit)

File: source/supported-hardware/whl-cfl-crb.rst (hunk 1 above)
Comment (raghavag) [reply source/supported-hardware/whl-cfl-crb.rst]: Corrected

File: source/supported-hardware/whl-cfl-crb.rst (hunk 3 above)
Comment (raghavag) [reply source/supported-hardware/whl-cfl-crb.rst]: Done

File: source/how-tos/boot-ubuntu.rst (hunk 4 above)
Comment (raghavag) [reply source/how-tos/boot-ubuntu.rst]: Done

File: source/supported-hardware/whl-cfl-crb.rst (hunk 2 above)
Comment (raghavag) [reply source/supported-hardware/whl-cfl-crb.rst]: Done

**Pull request merged**